client/components/external-link/docs/example.jsx

```diff
@@ -20,7 +20,7 @@ export default React.createClass( {
 					<a href="/devdocs/design/external-link">External Link</a>
 				</h2>
 				<Card>
-					<p><ExternalLink icon={ true } href="https://wordpress.org" >WordPress.org</ExternalLink></p>
+					<p><ExternalLink icon={ true } target="_blank" href="https://wordpress.org" >WordPress.org</ExternalLink></p>
```

Comment:
This shows for me as an error locally, which I'd expect. Not sure why it's not being flagged by CI.